modules/Microsoft.KeyVault/vaults/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,161 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
```

Comment:
Should we have a new empty line after each section? Currently no empty line after `Parameters`, one empty line after `Test execution`.

Reply:
We could - though I got to ask if this empty line vs. no empty (aka minor formatting) is worth blocking PRs.

Reply:
As per other comment above, I agree this shouldn't block the PRs, but wanted to at least mention it and keep track of it. The whole formatting and naming can be discussed separately and updated once all PRs, or the majority of them, will be in.